### PR TITLE
Updated Akka.Persistence.MySql to Akka 1.1.0

### PR DESCRIPTION
**Pull request by alexvaluyskiy**
Updated Akka.Persistence.MySql to Akka 1.1.0
Added Persistence.Query support

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,6 +75,7 @@ CREATE TABLE IF NOT EXISTS {your_journal_table_name} (
     created_at BIGINT NOT NULL,
     manifest VARCHAR(500) NOT NULL,
     payload BLOB NOT NULL,
+	tags VARCHAR(2000) NULL,
```
Comment (Horusiath) [inline README.md]: I've moved to varchar(100) in other implementations. I think, there's no reason to keep that number that high by default.

File: src/.nuget/NuGet.Dev.Config [hunk 2]
```diff
@@ -4,6 +4,9 @@
     <add key="disableSourceControlIntegration" value="true" />
   </solution>
   <packageSources>
-    <add key="NuGet official" value="https://www.nuget.org/api/v2/" /> <!-- Official package source  -->
+    <add key="NuGet official" value="https://www.nuget.org/api/v2/" />
+    <!-- Official package source  -->
+    <add key="Akka.NET Nightly Build" value="https://www.myget.org/F/akkadotnet/api/v2" />
```
Comment (Horusiath) [inline src/.nuget/NuGet.Dev.Config]: Do we need that source reference?

File: src/Akka.Persistence.MySql.Tests/MySqlJournalSpec.cs [hunk 3]
```diff
@@ -13,12 +18,12 @@ public class MySqlJournalSpec : JournalSpec
...
 
-        static MySqlJournalSpec() 
+        static MySqlJournalSpec()
         {
             var connectionString = ConfigurationManager.ConnectionStrings["TestDb"].ConnectionString;
 
-            var config = @"
-                akka.test.single-expect-default = 15s
+            SpecConfig = ConfigurationFactory.ParseString(@"
+                akka.test.single-expect-default = 20s
```
Comment (Horusiath) [inline src/Akka.Persistence.MySql.Tests/MySqlJournalSpec.cs]: Is that increase necessary?

**Comment by Horusiath** [comment]
Looks good.

File: src/.nuget/NuGet.Dev.Config (hunk 2 above)
Comment (alexvaluyskiy) [inline src/.nuget/NuGet.Dev.Config]: It is Dev config

**Comment by alexvaluyskiy** [comment]
@Horusiath ready to go

**Pull request merged**